AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_CubeMapCaptureAdded.py

```diff
@@ -87,22 +108,39 @@ def AtomEditorComponents_CubeMapCapture_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.removal_undo, cubemap_capture_entity.has_component(AtomComponentProperties.cube_map_capture()))
 
-        # 5. Delete Cubemap_Capture entity.
+        # 5. Set Specular IBL property: Very High
+        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL'), value=SPECULAR_IBL_QUALITY['Very High'])
+        Report.result(Tests.specular_ibl_property_set_to_very_high, cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL')) == SPECULAR_IBL_QUALITY['Very High'])
+
+        # 6. Set Specular IBL property: Very Low
+        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL'), value=SPECULAR_IBL_QUALITY['Very Low'])
+        Report.result(Tests.specular_ibl_property_set_to_very_low, cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL')) == SPECULAR_IBL_QUALITY['Very Low'])
+
+        # 7. Set Exposure property
+        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Exposure'), 16.0)
+        get_exposure_property = cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Exposure'))
+        Report.result(Tests.exposure_property_set, get_exposure_property == 16.0)
+
+        # 8. Set Capture Type property: Diffuse IBL
+        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Capture Type'), value=CUBEMAP_CAPTURE_TYPE['Diffuse ILB'])
+        Report.result(Tests.capture_type_property_set_to_Diffuse_ibl, cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Capture Type')) == CUBEMAP_CAPTURE_TYPE['Diffuse ILB'])
```

Comment:
you can for loop these values, but since there are only two it might not be of significant importance to do that.

Reply:
I was going to leave the loop off for this one since the default is Specular and that attribute is used in the previous tests. Switching to Diffuse is the last step.